hsm-gateway-applications/scripts/generate-hsm-user.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# define the CA setup
+CA_HOST=localhost
+CA_URL=${CA_HOST}:7054
+TLS_CERT='../../test-network/organizations/fabric-ca/org1/tls-cert.pem'
```

Comment:
For example: `TLS_CERT="${SCRIPT_DIR}/../../test-network/organizations/fabric-ca/org1/tls-cert.pem"`